modules/node_modules/@frogpond/ccci-stolaf-college/v1/streams/index.mjs

```diff
@@ -4,45 +4,62 @@ import moment from 'moment'
 
 const GET = mem(get, {maxAge: ONE_DAY})
 
-export async function getStreams(streamClass, dateFrom, dateTo) {
-	let params = {
+export async function getStreams({streamClass, sort, dateFrom, dateTo}) {
+	const params = {
 		class: streamClass,
-		sort: 'ascending',
+		sort: sort,
 		// eslint-disable-next-line camelcase
 		date_from: dateFrom,
 		// eslint-disable-next-line camelcase
 		date_to: dateTo,
 	}
 
-	let url = 'https://www.stolaf.edu/multimedia/api/collection'
+	const url = 'https://www.stolaf.edu/multimedia/api/collection'
 	const data = await GET(url, {json: true, query: params}).then(
 		resp => resp.body,
 	)
-	return data
+	const processed = data.results.map(stream => {
+		let {starttime} = stream
+		return {
+			...stream,
+			starttime: moment(starttime, 'YYYY-MM-DD').toISOString(),
+			date: moment(starttime, 'YYYY-MM-DD HH:mm'),
+		}
+	})
+
+	return processed
 }
 
 export async function upcoming(ctx) {
-	const {to} = ctx.query
-	const dateFrom = moment().format('YYYY-MM-DD')
-	const dateToDefault = moment()
-		.add(2, 'month')
-		.format('YYYY-MM-DD')
-
-	const dateToSelected = moment(to).format('YYYY-MM-DD')
-	const dateTo = to ? dateToSelected : dateToDefault
+	const {
+		date_from: dateFrom = moment().format('YYYY-MM-DD'),
```

Comment:
so… if this is coming from our app… let's change it to use camelCase instead of translating and adding eslint comments in both places